edx_event_bus_kafka/consumer/event_consumer.py

```diff
@@ -44,12 +49,17 @@ class KafkaEventConsumer:
     """
 
     def __init__(self, topic, group_id, signal):
-        self.topic = topic
-        self.group_id = group_id
-        self.signal = signal
-        self.consumer = self._create_consumer()
+        if confluent_kafka:
+            self.topic = topic
+            self.group_id = group_id
+            self.signal = signal
+            self.consumer = self._create_consumer()
+        else:
+            raise Exception('Library confluent-kafka not available. Cannot create event consumer.')
```

Comment:
I'd suggest raising this to an `if confluent_kafka is None:` so that we have an early-exit rather than wrapping the rest of the method in a conditional.